languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10249,7 +10249,8 @@ though, and has since been slightly modified)-->
                     <token postag="V.* ppa.*|V.* inf" postag_regexp="yes">
                         <exception postag="[NR].*|V ppa [fe] sp?" postag_regexp="yes"/></token>
                 </pattern>
-                <message>Voulez-vous écrire <suggestion>m'a</suggestion> ?</message>
+                <message>Une apostrophe est nécessaire afin de séparer le  pronom et le verbe avoir.</message>
```

Comment:
Is there actually a double space between "le" and "pronom", or is this just a GitHub artifact?